microsoft-azure-storage/src/com/microsoft/azure/storage/blob/BlobConstants.java

```diff
@@ -178,7 +178,7 @@
     /**
      * The maximum size, in bytes, of a blob before it must be separated into blocks
      */
-    // Note if this is updated then Constants.MAX_MARK_LENGTH needs to be as well.
+    //To copy small file without buffer operation overhead
```

Comment:
>//To copy small file without buffer operation overhead [](start = 4, length = 54)

can you please revert the change to this file?

Reply:
Updating comment back to it was?